mods/tuxemon/maps/taba_ba_br_1.tmx

```diff
@@ -0,0 +1,239 @@
+    <property name="act70" value="translated_dialog uhhjason"/>
+    <property name="act80" value="wait 2"/>
+    <property name="act90" value="npc_face omnigrunt7,down"/>
+    <property name="act91" value="wait 0.7"/>
+    <property name="act92" value="translated_dialog uhhjason2"/>
+    <property name="act93" value="wait 0.3"/>
+    <property name="act94" value="npc_move omnigrunt7,left 6"/>
+    <property name="act95" value="translated_dialog uhhjason3"/>
+    <property name="act96" value="wait 0.5"/>
+    <property name="act97" value="npc_move omnigrunt7, left 7"/>
+    <property name="act98" value="remove_npc omnigrunt7"/>
+    <property name="act99" value="unlock_controls"/>
+    <property name="act999" value="set_variable jason:gone"/>
+    <property name="cond1" value="is variable_set movealong:now"/>
+    <property name="cond2" value="not variable_set jason:gone"/>
+   </properties>
+  </object>
+  <object id="19" name="move to middle" type="event" x="176" y="48" width="16" height="160">
+   <properties>
+    <property name="act10" value="lock_controls"/>
+    <property name="act11" value="wait 0.3"/>
+    <property name="act12" value="translated_dialog challenger"/>
+    <property name="act20" value="pathfind player,9,7"/>
+    <property name="act21" value="npc_face player,left"/>
+    <property name="act30" value="set_variable battledone:start"/>
+    <property name="cond1" value="is player_at"/>
+    <property name="cond2" value="not variable_set battledone:yes"/>
+    <property name="cond3" value="not variable_set battledone:redo"/>
+   </properties>
+  </object>
+  <object id="20" name="start fight" type="event" x="144" y="16" width="16" height="16">
+   <properties>
+    <property name="act10" value="create_npc acolyte,0,7,,stand"/>
+    <property name="act11" value="npc_face acolyte,right"/>
+    <property name="act12" value="wait 1"/>
+    <property name="act20" value="pathfind acolyte,7,7"/>
+    <property name="act21" value="translated_dialog challenger2"/>
+    <property name="act22" value="wait 0.5"/>
+    <property name="act23" value="translated_dialog challenger3"/>
+    <property name="act30" value="wait 0.3"/>
+    <property name="act31" value="npc_face acolyte,up"/>
+    <property name="act32" value="wait 0.5"/>
+    <property name="act33" value="npc_face acolyte,down"/>
+    <property name="act34" value="wait 0.5"/>
+    <property name="act35" value="npc_face acolyte,right"/>
+    <property name="act36" value="translated_dialog challenger4"/>
+    <property name="act37" value="npc_face acolyte,left"/>
+    <property name="act38" value="wait 1"/>
+    <property name="act40" value="translated_dialog challenger5"/>
+    <property name="act41" value="npc_face acolyte,right"/>
+    <property name="act42" value="wait 0.4"/>
+    <property name="act43" value="translated_dialog challenger6"/>
+    <property name="act50" value="translated_dialog challenger7"/>
+    <property name="act60" value="start_battle acolyte"/>
```

Comment:
eventually you can add the line of the monster above start_battle
`    <property name="act55" value="add_monster chloragon,3,acolyte,80,100"/>`